lua/snacks/picker/select.lua

```diff
@@ -39,12 +40,23 @@ function M.select(items, opts, on_choice)
     },
     actions = {
       confirm = function(picker, item)
+        if completed then
+          return
+        end
+        completed = true
         picker:close()
         vim.schedule(function()
           on_choice(item and item.item, item and item.idx)
         end)
       end,
     },
+    on_close = function()
+      if not completed then
```

Comment:
shouldnt this be `if completed`?

Reply:
Ugh, yes. I refactored it to not use an early return, decided I didn't like it, and put it back. Except I forgot to update the conditional check 🤦‍♂️